pkgs/applications/networking/cluster/kube3d/default.nix

```diff
@@ -1,42 +1,51 @@
-{ stdenv, buildGoModule, fetchFromGitHub, installShellFiles }:
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
 
+let
+  k3sVersion = "1.18.9-k3s1";
+in
 buildGoModule rec {
   pname = "kube3d";
   version = "3.2.1";
-  k3sVersion = "1.18.9-k3s1";
 
-  excludedPackages = ''tools'';
+  excludedPackages = "tools";
 
   src = fetchFromGitHub {
-    owner  = "rancher";
-    repo   = "k3d";
-    rev    = "v${version}";
+    owner = "rancher";
+    repo = "k3d";
+    rev = "v${version}";
     sha256 = "1910vir9yc3xvb23260xljnfpznyhnk4gddy752rrndz7rk1b5c3";
   };
 
-  buildFlagsArray = ''
-    -ldflags=
-      -w -s
-      -X github.com/rancher/k3d/v3/version.Version=v${version}
-      -X github.com/rancher/k3d/v3/version.K3sVersion=v${k3sVersion}
-  '';
+  vendorSha256 = null;
 
   nativeBuildInputs = [ installShellFiles ];
 
+  buildFlagsArray = [
+    "-ldflags="
+    "-w"
+    "-s"
+    "-X github.com/rancher/k3d/v3/version.Version=v${version}"
+    "-X github.com/rancher/k3d/v3/version.K3sVersion=v${k3sVersion}"
+  ];
+
+  doCheck = false;
+
   postInstall = ''
     installShellCompletion --cmd k3d \
       --bash <($out/bin/k3d completion bash) \
       --fish <($out/bin/k3d completion fish) \
       --zsh <($out/bin/k3d completion zsh)
   '';
 
-  vendorSha256 = null;
-
-  doCheck = false;
-
-  meta = with stdenv.lib; {
+  meta = with lib; {
     homepage = "https://github.com/rancher/k3d";
-    description = "A helper to run k3s (Lightweight Kubernetes. 5 less than k8s) in a docker container";
+    description = "A helper to run k3s (Lightweight Kubernetes. 5 less than k8s) in a docker container - k3d";
```

Comment:
```suggestion
    description = "A helper to run k3s (Lightweight Kubernetes. 5 less than k8s) in a docker container";
```

Reply:
I added k3d to the short description because `nix search` uses it but doesn't look at `longDescription`

> `nix search` seems to only look at the short description so it's worth
including there too.

https://github.com/06kellyjac/nixpkgs/commit/83cd754e32c4651ce81887f7aca0336c53670407

Reply:
I think this is fine with me. Sound good to leave it @SuperSandro2000?